site/src/@ndlib/gatsby-theme-marble/components/Shared/SearchTools/SearchFilterBox/test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
Remove this test, it will never get run since tests are scoped to `@ndlib/gatsby-theme-marble

Reply:
Help me understand here, this file is in @ndlib/gatsby-theme-marble why won't it get run?

Reply:
All the places where we're running tests is with `yarn workspace @ndlib/gatsby-theme-marble test`. This is a shadow component in `/site` so it is outside of the testing scope. We could setup tests to run in both locations, but we currently have not.